.github/workflows/ci.yml

```diff
@@ -0,0 +1,74 @@
+name: ci
+
+"on":
+  push:
+    paths:
+      - ".github/workflows/ci.yml"
+      - "lib/**"
+      - "*.gemspec"
+      - "spec/**"
+      - "Rakefile"
+      - "Gemfile"
+      - ".rubocop.yml"
+  pull_request:
+    branches:
+      - master
+  create:
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - "2.7"
+        db:
+          - sqlite3
+          - mysql
+          - postgresql
+    env:
+      DB: ${{matrix.db}}
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install package dependencies
+        run: "[ -e $APT_DEPS ] || sudo apt-get install -y --no-install-recommends $APT_DEPS"
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{matrix.ruby}}
+      - name: Install latest bundler
+        run: |
+          gem install bundler --no-document
+      - name: Bundle install
+        run: bundle install --jobs 4 --retry 3
+      - name: Run all tests
+        env:
+          HANAMI_DATABASE_USERNAME: root
+          HANAMI_DATABASE_PASSWORD: root
+          HANAMI_DATABASE_HOST: 127.0.0.1
+          HANAMI_DATABASE_NAME: hanami_model
+        run: script/ci
+    services:
+      mysql:
+        image: mysql:8
+        env:
+          ALLOW_EMPTY_PASSWORD: true
+          MYSQL_AUTHENTICATION_PLUGIN: mysql_native_password
+          MYSQL_ROOT_PASSWORD: root
+          MYSQL_DATABASE: hanami_model
+        ports:
+          - 3306:3306
+        options: >-
+          --health-cmd="mysqladmin ping"
+          --health-interval=10s
+          --health-timeout=5s
+          --health-retries=3
+      postgres:
+        image: postgres:13
+        ports:
+          - 5432:5432
+        env:
+          POSTGRES_USER: root
+          POSTGRES_PASSWORD: root
+          POSTGRES_DB: hanami_model
```

Comment:
@jodosha instead of this, we can use GitHub's VM's installed postgresql in a simple run step after the `checkout` step

```yml
      - name: Set up PostgreSQL using VM's server
        run: |
          sudo systemctl start postgresql.service
          sudo -u postgres psql -c "CREATE USER runner WITH SUPERUSER PASSWORD 'runner'"
```

The user can be whatever is set in the config e.g. on my app I use this:
```yml
          DATABASE_URL: postgres://runner:runner@localhost/postgres-test-database
```

Reply:
However this isn't working 100% for me yet; postgresql still stalls..  I'm trying it out in parndt/model#2 if you want to look.

Ideally we could just use containers and not have stalls 😕